spk/salt-minion/Makefile

```diff
@@ -29,7 +28,6 @@ POST_STRIP_TARGET = salt-minion_extra_install
 
 # Service configuration
 CONF_DIR = src/conf
-SERVICE_USER = auto
```

Comment:
IMHO we need SERVICE_USER = auto for DSM 6, otherwise sc-salt is not created on installation.